solution-base/kubedb/certs.yaml

```diff
@@ -1,12 +1,12 @@
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```

Comment:
kubedb cert-manager is behind a paywall now so I do not believe they will update to 1.0 for the version we are using.